tests/test_full_example_spark.py

```diff
@@ -16,6 +16,7 @@
 from .linker_utils import _test_write_functionality, register_roc_data
 
 
+@mark_with_dialects_including("spark")
```

Comment:
marked spark because otherwise this ran as part of core tests, which slowed down the core/duckdb tests significantly